### PR TITLE
A word on translations

### PR DESCRIPTION
**Pull request by trasher**
See http://glpi-developer-documentation.readthedocs.io/en/feature-translations/devapi/translations.html

### CODE REVIEW

File: source/devapi/translations.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+What the system is capable to do:
+
+* replace variables (on LTR and RTL languages),
+* manage plural forms,
+* add context informations,
+* ...
+
+Here is the workflow used for translations:
+
+# Developers add string in the source code,
```
Comment (orthagh) [inline source/devapi/translations.rst]: seems this part is not render with new lines on rtfd

File: source/devapi/translations.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+# String are extracted to POT file,
+# POT file is sent to Transifex,
+# Transaltors translate,
```
Comment (orthagh) [inline source/devapi/translations.rst]: translators

File: source/devapi/translations.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+# Developers pull new translations from Transifex,
+# MO files iused by GLPI are generated.
```
Comment (orthagh) [inline source/devapi/translations.rst]: typo iused

File: source/devapi/translations.rst [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+Functions
+^^^^^^^^^
+
+There are several standard functions you will have to use in order to get translations. Remember the trnalsation domain will be `glpi` if not defined; so, for plugins specific translations, do not forget to set it!
```
Comment (orthagh) [inline source/devapi/translations.rst]: typo trnalsation

File: source/devapi/translations.rst [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+When you have a string to translate, but which rely on a count or something. You may as well use several functions, depending on the particular use case:
+
+* ``_n($sing, $plural, $nb, $domain='glpi')`` (what you will probably use the most frequently): give a string for singular form, another for plural form, and set current "count",
+* ``_sn($str, $domain='glpi')``: same as ``_n()`` but escape HTML entities,
+* ``_nx($ctx, $str, $domain='glpi')``: same as ``_n()`` but provide an extra context,
+
+Handle variables
+++++++++++++++++
+
+You may want to replace some parts of translations; for some reason. Let's say you would like to display current page on a total number of pages; you will use the `sprintf <http://php.net/manual/fr/function.sprintf.php>` method. This will allow you to make replacements; but without relying on arguments positions. For example:
```
Comment (orthagh) [inline source/devapi/translations.rst]: sprintf link doesn't render correctly, missing trailing _ ?

Comment (trasher) [inline source/devapi/translations.rst]: Well catched :)

**Comment by trasher** [comment]
Thanks!

**Pull request merged**